lib/oban/queue/producer.ex

```diff
@@ -18,6 +18,7 @@ defmodule Oban.Queue.Producer do
       :refresh_timer,
       dispatch_cooldown: 5,
       refresh_interval: :timer.seconds(30),
+      refresh_max_interval: :timer.seconds(60),
```

Comment:
How the interface for setting these two configurations should be?

Reply:
Let's not change this field. The implementation is an internal detail, we don't need to expose any controls over the jitter or imply how the interval is actually used.